shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
@mareklibra pls rename this file as well.
Seems this file should be named `workflow-outputs-schema.json` 
and the second file should be named in a singular form `workflow-output-schema.json`

Reply:
Done. What about `workflow-output-schema.json` and `workflow-output-schema.json`?

Reply:
sounds good. i was thinking of both, but then I noticed it isn't clear if the result contains the output or the opposite way around.
perhaps `workflow-result-schema.json` and `workflow-result-item-schema.json` so it is clear which schema is the main one.